core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -192,7 +192,7 @@ ServerStreamListener newServerStreamListener(ServerCall.Listener<ReqT> listener)
   }
 
   @Override
-  public Attributes attributes() {
+  public Attributes getAttributes() {
     return stream.attributes();
```

Comment:
I've just discovered that `(Client|Server)Stream` are inconsistent in the same way, in which case I think it's worth adjusting: https://github.com/grpc/grpc-java/pull/2616